src/sentry/api/endpoints/organization_stats.py

```diff
@@ -55,12 +55,19 @@ def get(self, request, organization):
                 user=request.user,
             )
 
-            project_list = []
-            for team in team_list:
-                project_list.extend(Project.objects.get_for_user(
-                    team=team,
-                    user=request.user,
-                ))
+            project_ids = request.GET.getlist('projectID')
```

Comment:
this isnt used yet -- i wanted to optimize the stats call but its a bit more complex than anticipated